<launch>
  <!-- Doc: http://wiki.ros.org/map_server?distro=kinetic -->

  <arg name="map_name" default="nerve_ant_demo" />

  <!-- launch map server -->
	<node name="map_server" pkg="map_server" type="map_server"
        args="$(find uml_hri_nerve_navigation)/resources/static_maps/2d/$(arg map_name).yaml"/>
</launch>
